hal/halx86/apic/apicp.h

```diff
@@ -54,6 +55,7 @@
     #define IrqlToTpr(Irql) (HalpIRQLtoTPR[Irql])
     #define IrqlToSoftVector(Irql) IrqlToTpr(Irql)
     #define TprToIrql(Tpr)  (HalVectorToIRQL[Tpr >> 4])
+    #define APIC_PROFILE_LEVEL HIGH_LEVEL
```

Comment:
I guess this is what we would want,
but can code outside APIC HAL cope with this change?
https://jira.reactos.org/browse/CORE-20093?focusedId=144726&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-144726

Reply:
The current kernel code can deal with it.